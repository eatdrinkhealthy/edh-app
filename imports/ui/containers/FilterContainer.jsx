// @flow
import React from "react";
import Filter from "../components/Filter";
import FILTER_LIST from "../../api/filters";

const FilterContainer = () => (
  <Filter filterList={FILTER_LIST} />
);

export default FilterContainer;
