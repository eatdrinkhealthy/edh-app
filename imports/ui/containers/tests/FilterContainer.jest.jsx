/* eslint-env jest */
/* eslint-disable func-names, prefer-arrow-callback, no-unused-expressions */
/* eslint-disable import/no-extraneous-dependencies */

import React from "react";
import renderer from "react-test-renderer";
import FilterContainer from "../FilterContainer";

describe("<FilterContainer />", function () {
  it("matches render snapshot", function () {
    const tree = renderer.create(<FilterContainer />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
