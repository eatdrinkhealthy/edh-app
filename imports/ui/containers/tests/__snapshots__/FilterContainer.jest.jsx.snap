exports[`<FilterContainer /> matches render snapshot 1`] = `
<div>
  <div
    className="filter-header">
    <h4>
      Filter
    </h4>
    <a
      onClick={[Function]}
      style={Object {}}>
      <div
        className="close-filter toggle-filter" />
    </a>
  </div>
  <div
    className="filter">
    <div>
      <span
        className="react-toggle-label">
        Juice Bars
      </span>
      <span
        className="react-toggle">
        <div
          className="react-toggle"
          onClick={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}>
          <div
            className="react-toggle-track">
            <div
              className="react-toggle-track-check">
              <svg
                height="11"
                viewBox="0 0 14 11"
                width="14">
                <title>
                  switch-check
                </title>
                <path
                  d="M11.264 0L5.26 6.004 2.103 2.847 0 4.95l5.26 5.26 8.108-8.107L11.264 0"
                  fill="#fff"
                  fillRule="evenodd" />
              </svg>
            </div>
            <div
              className="react-toggle-track-x">
              <svg
                height="10"
                viewBox="0 0 10 10"
                width="10">
                <title>
                  switch-x
                </title>
                <path
                  d="M9.9 2.12L7.78 0 4.95 2.828 2.12 0 0 2.12l2.83 2.83L0 7.776 2.123 9.9 4.95 7.07 7.78 9.9 9.9 7.776 7.072 4.95 9.9 2.12"
                  fill="#fff"
                  fillRule="evenodd" />
              </svg>
            </div>
          </div>
          <div
            className="react-toggle-thumb" />
          <input
            className="react-toggle-screenreader-only"
            id="juiceBar"
            onBlur={[Function]}
            onFocus={[Function]}
            type="checkbox" />
        </div>
      </span>
    </div>
    <div>
      <span
        className="react-toggle-label">
        Vegan / Vegetarian
      </span>
      <span
        className="react-toggle">
        <div
          className="react-toggle"
          onClick={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}>
          <div
            className="react-toggle-track">
            <div
              className="react-toggle-track-check">
              <svg
                height="11"
                viewBox="0 0 14 11"
                width="14">
                <title>
                  switch-check
                </title>
                <path
                  d="M11.264 0L5.26 6.004 2.103 2.847 0 4.95l5.26 5.26 8.108-8.107L11.264 0"
                  fill="#fff"
                  fillRule="evenodd" />
              </svg>
            </div>
            <div
              className="react-toggle-track-x">
              <svg
                height="10"
                viewBox="0 0 10 10"
                width="10">
                <title>
                  switch-x
                </title>
                <path
                  d="M9.9 2.12L7.78 0 4.95 2.828 2.12 0 0 2.12l2.83 2.83L0 7.776 2.123 9.9 4.95 7.07 7.78 9.9 9.9 7.776 7.072 4.95 9.9 2.12"
                  fill="#fff"
                  fillRule="evenodd" />
              </svg>
            </div>
          </div>
          <div
            className="react-toggle-thumb" />
          <input
            className="react-toggle-screenreader-only"
            id="veganVegRestaurant"
            onBlur={[Function]}
            onFocus={[Function]}
            type="checkbox" />
        </div>
      </span>
    </div>
    <div>
      <span
        className="react-toggle-label">
        Gluten Free
      </span>
      <span
        className="react-toggle">
        <div
          className="react-toggle"
          onClick={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}>
          <div
            className="react-toggle-track">
            <div
              className="react-toggle-track-check">
              <svg
                height="11"
                viewBox="0 0 14 11"
                width="14">
                <title>
                  switch-check
                </title>
                <path
                  d="M11.264 0L5.26 6.004 2.103 2.847 0 4.95l5.26 5.26 8.108-8.107L11.264 0"
                  fill="#fff"
                  fillRule="evenodd" />
              </svg>
            </div>
            <div
              className="react-toggle-track-x">
              <svg
                height="10"
                viewBox="0 0 10 10"
                width="10">
                <title>
                  switch-x
                </title>
                <path
                  d="M9.9 2.12L7.78 0 4.95 2.828 2.12 0 0 2.12l2.83 2.83L0 7.776 2.123 9.9 4.95 7.07 7.78 9.9 9.9 7.776 7.072 4.95 9.9 2.12"
                  fill="#fff"
                  fillRule="evenodd" />
              </svg>
            </div>
          </div>
          <div
            className="react-toggle-thumb" />
          <input
            className="react-toggle-screenreader-only"
            id="glutenFree"
            onBlur={[Function]}
            onFocus={[Function]}
            type="checkbox" />
        </div>
      </span>
    </div>
    <div>
      <span
        className="react-toggle-label">
        Salad Places
      </span>
      <span
        className="react-toggle">
        <div
          className="react-toggle"
          onClick={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}>
          <div
            className="react-toggle-track">
            <div
              className="react-toggle-track-check">
              <svg
                height="11"
                viewBox="0 0 14 11"
                width="14">
                <title>
                  switch-check
                </title>
                <path
                  d="M11.264 0L5.26 6.004 2.103 2.847 0 4.95l5.26 5.26 8.108-8.107L11.264 0"
                  fill="#fff"
                  fillRule="evenodd" />
              </svg>
            </div>
            <div
              className="react-toggle-track-x">
              <svg
                height="10"
                viewBox="0 0 10 10"
                width="10">
                <title>
                  switch-x
                </title>
                <path
                  d="M9.9 2.12L7.78 0 4.95 2.828 2.12 0 0 2.12l2.83 2.83L0 7.776 2.123 9.9 4.95 7.07 7.78 9.9 9.9 7.776 7.072 4.95 9.9 2.12"
                  fill="#fff"
                  fillRule="evenodd" />
              </svg>
            </div>
          </div>
          <div
            className="react-toggle-thumb" />
          <input
            className="react-toggle-screenreader-only"
            id="saladPlace"
            onBlur={[Function]}
            onFocus={[Function]}
            type="checkbox" />
        </div>
      </span>
    </div>
    <div>
      <span
        className="react-toggle-label">
        Bakeries
      </span>
      <span
        className="react-toggle">
        <div
          className="react-toggle"
          onClick={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}>
          <div
            className="react-toggle-track">
            <div
              className="react-toggle-track-check">
              <svg
                height="11"
                viewBox="0 0 14 11"
                width="14">
                <title>
                  switch-check
                </title>
                <path
                  d="M11.264 0L5.26 6.004 2.103 2.847 0 4.95l5.26 5.26 8.108-8.107L11.264 0"
                  fill="#fff"
                  fillRule="evenodd" />
              </svg>
            </div>
            <div
              className="react-toggle-track-x">
              <svg
                height="10"
                viewBox="0 0 10 10"
                width="10">
                <title>
                  switch-x
                </title>
                <path
                  d="M9.9 2.12L7.78 0 4.95 2.828 2.12 0 0 2.12l2.83 2.83L0 7.776 2.123 9.9 4.95 7.07 7.78 9.9 9.9 7.776 7.072 4.95 9.9 2.12"
                  fill="#fff"
                  fillRule="evenodd" />
              </svg>
            </div>
          </div>
          <div
            className="react-toggle-thumb" />
          <input
            className="react-toggle-screenreader-only"
            id="bakery"
            onBlur={[Function]}
            onFocus={[Function]}
            type="checkbox" />
        </div>
      </span>
    </div>
    <div>
      <span
        className="react-toggle-label">
        Caf√©s
      </span>
      <span
        className="react-toggle">
        <div
          className="react-toggle"
          onClick={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}>
          <div
            className="react-toggle-track">
            <div
              className="react-toggle-track-check">
              <svg
                height="11"
                viewBox="0 0 14 11"
                width="14">
                <title>
                  switch-check
                </title>
                <path
                  d="M11.264 0L5.26 6.004 2.103 2.847 0 4.95l5.26 5.26 8.108-8.107L11.264 0"
                  fill="#fff"
                  fillRule="evenodd" />
              </svg>
            </div>
            <div
              className="react-toggle-track-x">
              <svg
                height="10"
                viewBox="0 0 10 10"
                width="10">
                <title>
                  switch-x
                </title>
                <path
                  d="M9.9 2.12L7.78 0 4.95 2.828 2.12 0 0 2.12l2.83 2.83L0 7.776 2.123 9.9 4.95 7.07 7.78 9.9 9.9 7.776 7.072 4.95 9.9 2.12"
                  fill="#fff"
                  fillRule="evenodd" />
              </svg>
            </div>
          </div>
          <div
            className="react-toggle-thumb" />
          <input
            className="react-toggle-screenreader-only"
            id="cafe"
            onBlur={[Function]}
            onFocus={[Function]}
            type="checkbox" />
        </div>
      </span>
    </div>
    <div>
      <span
        className="react-toggle-label">
        Coffee Shops
      </span>
      <span
        className="react-toggle">
        <div
          className="react-toggle"
          onClick={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}>
          <div
            className="react-toggle-track">
            <div
              className="react-toggle-track-check">
              <svg
                height="11"
                viewBox="0 0 14 11"
                width="14">
                <title>
                  switch-check
                </title>
                <path
                  d="M11.264 0L5.26 6.004 2.103 2.847 0 4.95l5.26 5.26 8.108-8.107L11.264 0"
                  fill="#fff"
                  fillRule="evenodd" />
              </svg>
            </div>
            <div
              className="react-toggle-track-x">
              <svg
                height="10"
                viewBox="0 0 10 10"
                width="10">
                <title>
                  switch-x
                </title>
                <path
                  d="M9.9 2.12L7.78 0 4.95 2.828 2.12 0 0 2.12l2.83 2.83L0 7.776 2.123 9.9 4.95 7.07 7.78 9.9 9.9 7.776 7.072 4.95 9.9 2.12"
                  fill="#fff"
                  fillRule="evenodd" />
              </svg>
            </div>
          </div>
          <div
            className="react-toggle-thumb" />
          <input
            className="react-toggle-screenreader-only"
            id="coffeeShop"
            onBlur={[Function]}
            onFocus={[Function]}
            type="checkbox" />
        </div>
      </span>
    </div>
    <div>
      <span
        className="react-toggle-label">
        Restaurants
      </span>
      <span
        className="react-toggle">
        <div
          className="react-toggle"
          onClick={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}>
          <div
            className="react-toggle-track">
            <div
              className="react-toggle-track-check">
              <svg
                height="11"
                viewBox="0 0 14 11"
                width="14">
                <title>
                  switch-check
                </title>
                <path
                  d="M11.264 0L5.26 6.004 2.103 2.847 0 4.95l5.26 5.26 8.108-8.107L11.264 0"
                  fill="#fff"
                  fillRule="evenodd" />
              </svg>
            </div>
            <div
              className="react-toggle-track-x">
              <svg
                height="10"
                viewBox="0 0 10 10"
                width="10">
                <title>
                  switch-x
                </title>
                <path
                  d="M9.9 2.12L7.78 0 4.95 2.828 2.12 0 0 2.12l2.83 2.83L0 7.776 2.123 9.9 4.95 7.07 7.78 9.9 9.9 7.776 7.072 4.95 9.9 2.12"
                  fill="#fff"
                  fillRule="evenodd" />
              </svg>
            </div>
          </div>
          <div
            className="react-toggle-thumb" />
          <input
            className="react-toggle-screenreader-only"
            id="restaurant"
            onBlur={[Function]}
            onFocus={[Function]}
            type="checkbox" />
        </div>
      </span>
    </div>
    <div>
      <span
        className="react-toggle-label">
        Farmers Markets
      </span>
      <span
        className="react-toggle">
        <div
          className="react-toggle"
          onClick={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}>
          <div
            className="react-toggle-track">
            <div
              className="react-toggle-track-check">
              <svg
                height="11"
                viewBox="0 0 14 11"
                width="14">
                <title>
                  switch-check
                </title>
                <path
                  d="M11.264 0L5.26 6.004 2.103 2.847 0 4.95l5.26 5.26 8.108-8.107L11.264 0"
                  fill="#fff"
                  fillRule="evenodd" />
              </svg>
            </div>
            <div
              className="react-toggle-track-x">
              <svg
                height="10"
                viewBox="0 0 10 10"
                width="10">
                <title>
                  switch-x
                </title>
                <path
                  d="M9.9 2.12L7.78 0 4.95 2.828 2.12 0 0 2.12l2.83 2.83L0 7.776 2.123 9.9 4.95 7.07 7.78 9.9 9.9 7.776 7.072 4.95 9.9 2.12"
                  fill="#fff"
                  fillRule="evenodd" />
              </svg>
            </div>
          </div>
          <div
            className="react-toggle-thumb" />
          <input
            className="react-toggle-screenreader-only"
            id="farmersMarket"
            onBlur={[Function]}
            onFocus={[Function]}
            type="checkbox" />
        </div>
      </span>
    </div>
    <div>
      <span
        className="react-toggle-label">
        Health Food Stores
      </span>
      <span
        className="react-toggle">
        <div
          className="react-toggle"
          onClick={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}>
          <div
            className="react-toggle-track">
            <div
              className="react-toggle-track-check">
              <svg
                height="11"
                viewBox="0 0 14 11"
                width="14">
                <title>
                  switch-check
                </title>
                <path
                  d="M11.264 0L5.26 6.004 2.103 2.847 0 4.95l5.26 5.26 8.108-8.107L11.264 0"
                  fill="#fff"
                  fillRule="evenodd" />
              </svg>
            </div>
            <div
              className="react-toggle-track-x">
              <svg
                height="10"
                viewBox="0 0 10 10"
                width="10">
                <title>
                  switch-x
                </title>
                <path
                  d="M9.9 2.12L7.78 0 4.95 2.828 2.12 0 0 2.12l2.83 2.83L0 7.776 2.123 9.9 4.95 7.07 7.78 9.9 9.9 7.776 7.072 4.95 9.9 2.12"
                  fill="#fff"
                  fillRule="evenodd" />
              </svg>
            </div>
          </div>
          <div
            className="react-toggle-thumb" />
          <input
            className="react-toggle-screenreader-only"
            id="healthFoodStore"
            onBlur={[Function]}
            onFocus={[Function]}
            type="checkbox" />
        </div>
      </span>
    </div>
    <div>
      <span
        className="react-toggle-label">
        Organic Grocery Stores
      </span>
      <span
        className="react-toggle">
        <div
          className="react-toggle"
          onClick={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}>
          <div
            className="react-toggle-track">
            <div
              className="react-toggle-track-check">
              <svg
                height="11"
                viewBox="0 0 14 11"
                width="14">
                <title>
                  switch-check
                </title>
                <path
                  d="M11.264 0L5.26 6.004 2.103 2.847 0 4.95l5.26 5.26 8.108-8.107L11.264 0"
                  fill="#fff"
                  fillRule="evenodd" />
              </svg>
            </div>
            <div
              className="react-toggle-track-x">
              <svg
                height="10"
                viewBox="0 0 10 10"
                width="10">
                <title>
                  switch-x
                </title>
                <path
                  d="M9.9 2.12L7.78 0 4.95 2.828 2.12 0 0 2.12l2.83 2.83L0 7.776 2.123 9.9 4.95 7.07 7.78 9.9 9.9 7.776 7.072 4.95 9.9 2.12"
                  fill="#fff"
                  fillRule="evenodd" />
              </svg>
            </div>
          </div>
          <div
            className="react-toggle-thumb" />
          <input
            className="react-toggle-screenreader-only"
            id="organicGrocery"
            onBlur={[Function]}
            onFocus={[Function]}
            type="checkbox" />
        </div>
      </span>
    </div>
    <div>
      <span
        className="react-toggle-label">
        Grocery Stores
      </span>
      <span
        className="react-toggle">
        <div
          className="react-toggle"
          onClick={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}>
          <div
            className="react-toggle-track">
            <div
              className="react-toggle-track-check">
              <svg
                height="11"
                viewBox="0 0 14 11"
                width="14">
                <title>
                  switch-check
                </title>
                <path
                  d="M11.264 0L5.26 6.004 2.103 2.847 0 4.95l5.26 5.26 8.108-8.107L11.264 0"
                  fill="#fff"
                  fillRule="evenodd" />
              </svg>
            </div>
            <div
              className="react-toggle-track-x">
              <svg
                height="10"
                viewBox="0 0 10 10"
                width="10">
                <title>
                  switch-x
                </title>
                <path
                  d="M9.9 2.12L7.78 0 4.95 2.828 2.12 0 0 2.12l2.83 2.83L0 7.776 2.123 9.9 4.95 7.07 7.78 9.9 9.9 7.776 7.072 4.95 9.9 2.12"
                  fill="#fff"
                  fillRule="evenodd" />
              </svg>
            </div>
          </div>
          <div
            className="react-toggle-thumb" />
          <input
            className="react-toggle-screenreader-only"
            id="grocery"
            onBlur={[Function]}
            onFocus={[Function]}
            type="checkbox" />
        </div>
      </span>
    </div>
    <div>
      <span
        className="react-toggle-label">
        Supermarkets
      </span>
      <span
        className="react-toggle">
        <div
          className="react-toggle"
          onClick={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}>
          <div
            className="react-toggle-track">
            <div
              className="react-toggle-track-check">
              <svg
                height="11"
                viewBox="0 0 14 11"
                width="14">
                <title>
                  switch-check
                </title>
                <path
                  d="M11.264 0L5.26 6.004 2.103 2.847 0 4.95l5.26 5.26 8.108-8.107L11.264 0"
                  fill="#fff"
                  fillRule="evenodd" />
              </svg>
            </div>
            <div
              className="react-toggle-track-x">
              <svg
                height="10"
                viewBox="0 0 10 10"
                width="10">
                <title>
                  switch-x
                </title>
                <path
                  d="M9.9 2.12L7.78 0 4.95 2.828 2.12 0 0 2.12l2.83 2.83L0 7.776 2.123 9.9 4.95 7.07 7.78 9.9 9.9 7.776 7.072 4.95 9.9 2.12"
                  fill="#fff"
                  fillRule="evenodd" />
              </svg>
            </div>
          </div>
          <div
            className="react-toggle-thumb" />
          <input
            className="react-toggle-screenreader-only"
            id="supermarket"
            onBlur={[Function]}
            onFocus={[Function]}
            type="checkbox" />
        </div>
      </span>
    </div>
    <div>
      <span
        className="react-toggle-label">
        Fruit & Vegetable Stores
      </span>
      <span
        className="react-toggle">
        <div
          className="react-toggle"
          onClick={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}>
          <div
            className="react-toggle-track">
            <div
              className="react-toggle-track-check">
              <svg
                height="11"
                viewBox="0 0 14 11"
                width="14">
                <title>
                  switch-check
                </title>
                <path
                  d="M11.264 0L5.26 6.004 2.103 2.847 0 4.95l5.26 5.26 8.108-8.107L11.264 0"
                  fill="#fff"
                  fillRule="evenodd" />
              </svg>
            </div>
            <div
              className="react-toggle-track-x">
              <svg
                height="10"
                viewBox="0 0 10 10"
                width="10">
                <title>
                  switch-x
                </title>
                <path
                  d="M9.9 2.12L7.78 0 4.95 2.828 2.12 0 0 2.12l2.83 2.83L0 7.776 2.123 9.9 4.95 7.07 7.78 9.9 9.9 7.776 7.072 4.95 9.9 2.12"
                  fill="#fff"
                  fillRule="evenodd" />
              </svg>
            </div>
          </div>
          <div
            className="react-toggle-thumb" />
          <input
            className="react-toggle-screenreader-only"
            id="fruitVegStore"
            onBlur={[Function]}
            onFocus={[Function]}
            type="checkbox" />
        </div>
      </span>
    </div>
    <div>
      <span
        className="react-toggle-label">
        Markets
      </span>
      <span
        className="react-toggle">
        <div
          className="react-toggle"
          onClick={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}>
          <div
            className="react-toggle-track">
            <div
              className="react-toggle-track-check">
              <svg
                height="11"
                viewBox="0 0 14 11"
                width="14">
                <title>
                  switch-check
                </title>
                <path
                  d="M11.264 0L5.26 6.004 2.103 2.847 0 4.95l5.26 5.26 8.108-8.107L11.264 0"
                  fill="#fff"
                  fillRule="evenodd" />
              </svg>
            </div>
            <div
              className="react-toggle-track-x">
              <svg
                height="10"
                viewBox="0 0 10 10"
                width="10">
                <title>
                  switch-x
                </title>
                <path
                  d="M9.9 2.12L7.78 0 4.95 2.828 2.12 0 0 2.12l2.83 2.83L0 7.776 2.123 9.9 4.95 7.07 7.78 9.9 9.9 7.776 7.072 4.95 9.9 2.12"
                  fill="#fff"
                  fillRule="evenodd" />
              </svg>
            </div>
          </div>
          <div
            className="react-toggle-thumb" />
          <input
            className="react-toggle-screenreader-only"
            id="market"
            onBlur={[Function]}
            onFocus={[Function]}
            type="checkbox" />
        </div>
      </span>
    </div>
  </div>
</div>
`;
